package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "vanilla-framework",
-  "version": "4.18.0",
+  "version": "4.18.1",
```

Sorry for the version bumping confusion. It's more on us and unclear guidelines about it.
4.18 is still in development (in main branch), not released yet, so no need to bump the version. Thanks!

```suggestion
  "version": "4.18.0",
```